obspy/io/arclink/inventory.py

```diff
@@ -0,0 +1,929 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+ObsPy implementation for parsing the arclink inventory format
+to an Inventory object.
+
+This is a modified version of obspy.io.stationxml and obspy.io.sc3ml.
+
+:author:
+    Mathijs Koymans (koymans@knmi.nl), 29.2016 - [Jollyfant@GitHub]
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import inspect
+import re
+import math
+import os
+import warnings
```

Comment:
Please sort the stdlib imports alphabetically and move the obspy import down to the other obspy imports.